README.md

```diff
@@ -134,4 +134,4 @@ See [LICENSE](LICENSE) for details.
 
 ## Framework Evaluation
 
-See [FRAMEWORK_EVALUATION.md](FRAMEWORK_EVALUATION.md) for a comprehensive analysis of various cross-platform framework options and why Compose Multiplatform was selected.
\ No newline at end of file
+See [FRAMEWORK_EVALUATION.md](docs/FRAMEWORK_EVALUATION.md) for a comprehensive analysis of various cross-platform framework options and why Compose Multiplatform was selected.
```

Comment:
No need for this whole `## Framework Evaluation` section since it is already mentioned in line 103 as `- [Framework Evaluation](docs/FRAMEWORK_EVALUATION.md) - Why Compose Multiplatform was chosen`